centreon/src/Core/Infrastructure/Common/Api/Router.php

```diff
@@ -159,4 +159,19 @@ public function warmUp(string $cacheDir)
     {
         return [];
     }
+
+    /**
+     * Create a href to a legacy page.
+     *
+     * @param int $topologyPage
+     * @param string|null $options
+     *
+     * @return string
+     */
+    public function computeLegacyHref(int $topologyPage, ?string $options = null): string
```

Comment:
To be consistent with the basic 'generate' method
```suggestion
    public function generateLegacyHref(int $topologyPage, ?string $options = null): string
```